src/pages/Material/index.js

```diff
@@ -11,61 +11,61 @@ const Material = () => {
   const [loans, setLoans] = useState([]);
 
   useEffect(() => {
-    api
-      .get('/materials')
-      .then((response) => {
-        if (response.status === 200) {
-          setMaterials(response.data);
-        }
-      })
-      .catch((error) => {
-        console.log(error);
-      });
+    const getMaterials = async () => {
+      const { data } = await api.get('/materials');
+      setMaterials(data);
+    };
+    getMaterials();
   }, []);
 
-  useEffect(async () => {
-    const { data } = await api.get('/loans');
-
-    const formattedLoans = data
-      .map((l) => {
-        if (l.return_time) {
-          l.return_time = formatRelative(
-            toDate(parseISO(l.return_time)),
-            new Date(),
-            {
-              locale: pt,
-            }
-          );
-        } else {
-          l.loan_time = formatRelative(
-            toDate(parseISO(l.loan_time)),
-            new Date(),
-            {
-              locale: pt,
+  useEffect(() => {
+    api.get('/loans').then(({ data }) => {
+      if (data.length !== 0) {
+        const formattedLoans = data
+          .map((l) => {
+            if (l.return_time) {
+              l.return_time = formatRelative(
+                toDate(parseISO(l.return_time)),
+                new Date(),
+                {
+                  locale: pt,
+                }
+              );
+            } else {
+              l.loan_time = formatRelative(
+                toDate(parseISO(l.loan_time)),
+                new Date(),
+                {
+                  locale: pt,
+                }
+              );
```

Comment:
Well, when we are using async into useEffect, we can't use async before callback. To do that, we have to create an async function inside of useEffect, then we call it normaly without await. For instance:
```js
useEffect(() => {
  async function fetchData() { /* piece of code */ }
  fetchData();
}, []);
```

Reply:
> Well, when we are using async into useEffect, we can't use async before callback. To do that, we have to create an async function inside of useEffect, then we call it normaly without await. For instance:
> 
> ```js
> useEffect(() => {
>   async function fetchData() { /* piece of code */ }
>   fetchData();
> }, []);
> ```

Yeah, I got that when I was coding, but when returning the variable with the await result from the function, I actually get it an promise pending, so I would need to resolve it again, so I make in the old way and put a then there lol